tests/tests.rs

```diff
@@ -1911,3 +1911,27 @@ fn serde() {
         "invalid value: integer `-1`, expected u128",
     );
 }
+
+#[cfg(all(feature = "borsh", feature = "std"))]
+#[test]
+fn borsh() {
```

Comment:
We should add test-borsh.yml to ensure this is automatically tested on github (see e.g. test-serde.yml)

Reply:
ok, will add.